eclipse-scout-core/src/tabbox/SimpleTabAreaLayout.ts

```diff
@@ -24,12 +25,35 @@ export class SimpleTabAreaLayout extends AbstractLayout {
     this._$overflowTab = null;
     this._overflowTabsIndizes = [];
 
-    this.tabWidth = null;
-    this.tabMinWidth = null;
-    this.overflowTabItemWidth = null;
+    this.tabSize = null;
+    this.tabMinSize = null;
+    this.overflowTabItemSize = null;
+  }
+
+  protected _hasHorizontalTabs(): boolean {
```

Comment:
```suggestion
  protected _calculateHorizontal(): boolean {
```

Call this method only in `_initSizes`, everywhere else use `this._horizontal`.